packages/rpc-graphql/src/schema/instruction.ts

```diff
@@ -553,6 +553,21 @@ export const instructionTypeDefs = /* GraphQL */ `
         maximumFee: Int
     }
 
+    """
+    SplToken-2022: TransferCheckedWithFee instruction
+    """
+    type SplTokenTransferCheckedWithFee implements TransactionInstruction {
+        programId: Address
+        authority: Account
+        destination: Account
+        feeAmount: TokenAmount
+        mint: Account
+        source: Account
+        tokenAmount: TokenAmount
+        multisigAuthority: Account
+        signers: [Address]
```

Comment:
Here we want to use `[Account]`, but we'll need to roll a new resolver to do multiple accounts. I _think_ it should be fairly trivial to do, but I can roll it soon if you wanted to keep chugging on the extensions. We can flip this to `[Account]` when I do. Unless you want to give it a whirl?

Reply:
Should I start by creating a new function `resolveAccounts` function with below signature 


```typescript

export const resolveAccounts = (fieldName?: string) => {
   return async (
       parent: { [x: string]: Array<Address> },
       args: { address?: Address; commitment?: Commitment; minContextSlot?: Slot },
       context: RpcGraphQLContext,
       info: GraphQLResolveInfo): Promise<AccountResult | null> => {

      const addresses = fieldName ? parent[fieldName] : null; // This will be an Array of address. 
      // Below handle logic for multiple address

}
}
```

Changing the existing function to return multiple address could cause things to break.

Reply:
Let me hack around on this for a minute. I think there's actually a simple way to do this with the way we've laid down the batch loader. I'll circle back here ASAP; working on it now.

Reply:
But yes, the interface you've posted is correct. I think we might be able to just roll that function signature as you've posted it, but then simply "map" the response from `resolveAccount`.

Reply:
I'm going to roll a new `resolveMultipleAccounts` resolver, as well as a `multipleAccounts` root query, as I mentioned [here](https://github.com/solana-labs/solana-web3.js/pull/2579#discussion_r1585502257). So go ahead and leave this as `[Address]` for now.